.gitignore

```diff
@@ -1,4 +1,5 @@
 /vendor
+.golangci.yml
```

Comment:
But we don't have `make` here, so maybe add it? Although manual downloading is always an option.

Reply:
I don't want to add a complete Makefile where it's not used now. But if you're to copy the file it'd be show in `git status` while it should be ignored now to avoid accidental addition to the repository again.

Reply:
OK, we choose manual downloading.